apps/warehouse/src/routers/dashboard/role/modules/_id/edit-role.tsx

```diff
@@ -0,0 +1,90 @@
+import { TRole } from '@fms/entities';
+import {
+  ControlledFieldCheckbox,
+  ControlledFieldSelect,
+  DataTable,
+} from '@fms/organisms';
+import { ColumnDef } from '@tanstack/react-table';
+import { FC, ReactElement } from 'react';
+import { useForm } from 'react-hook-form';
+
+export const EditRole: FC = (): ReactElement => {
+  const roles = [
+    { label: 'Super Admin', value: 'super admin' },
+    { label: 'Admin', value: 'admin' },
+  ];
+  const { control } = useForm();
+  const dataRoles = [
+    { id: 1, name: 'Warehouse', permission: 'read' },
+    { id: 2, name: 'Back Office', permission: 'read' },
+    { id: 3, name: 'Client', permission: 'read' },
+  ];
+  const columns: ColumnDef<TRole>[] = [
+    {
+      header: 'Nama Modul',
+      accessorKey: 'name',
+      cell: ({ cell }) => <p>{cell.row.original.name}</p>,
+    },
+    {
+      header: 'Jenis Permission',
+      accessorKey: 'permission',
+      cell({ row }) {
+        return (
+          <form className="flex gap-x-1">
+            <ControlledFieldCheckbox
+              name="permission"
```

Comment:
di map dari array of permission ieu kuduna

Reply:
noted